### ISSUE TITLE
[BUG] -  Team Fact Sheet Symbology not displaying

### ISSUE THREAD

**Issue by pjdohertygis** [pjdohertygis/SARCOP#267]
### Which app?

Dashboards

### Describe the Device, Operating System, and Browser

Windows PC 
Windows 11 
Chrome Version 125.0.6422.176 (Official Build) (64-bit)
May be occurring in Edge as well but unconfirmed.

### Describe the bug

Symbology in Team Fact Sheet Dashboard occasionally not displaying symbols. So far it appears to be worse when used embedded in Experience Builder - *however* the issue has been seen elsewhere in the FEMA NRCC Dashboard, in web maps, and on the feature layer view visualization tab itself. No helpful errors in the Console, just "symptoms".

Here is an example with the Dashboard embedded in an HTML frame not displaying (note the symbols do show up in the list element)
https://experience.arcgis.com/experience/9051a07ffd1947bbb395e9f9b4088f85/page/Team-Fact-Sheet-Dashboard/
![image](https://github.com/pjdohertygis/SARCOP/assets/4256979/786ba55a-ece1-441d-834c-6c4a997c898c)

Here is an example with the Dashboard in Chrome not displaying (note the symbols do show up in the list element)
https://www.arcgis.com/apps/dashboards/f1bdf94be2c94c8c96e0b8654c7fcd9a
![image](https://github.com/pjdohertygis/SARCOP/assets/4256979/071c408d-f0cb-46eb-bafc-9c7f210f01ef)

Interestingly, when I click on the list, the map zooms and the pop-up opens up now problem 
![image](https://github.com/pjdohertygis/SARCOP/assets/4256979/484b1c5f-2637-4146-8e66-b39b3781a3ab)

Here is an example of the web map showing the symbols properly
https://www.arcgis.com/apps/mapviewer/index.html?webmap=489f5e3ab9044b24bc4325e4ad27257c
![image](https://github.com/pjdohertygis/SARCOP/assets/4256979/fa9490e6-1a77-465c-a017-b3d29e8a40c9)

Here is an example of the feature layer view visualization tab 
https://www.arcgis.com/home/item.html?id=426e88e2b827464abd415041e1af3ef6#visualize
![image](https://github.com/pjdohertygis/SARCOP/assets/4256979/f5a17744-1499-4836-9786-d12c44fe3b1c)

Also tried using Google Chrome Incognito (signed in as blueist_giss) and same issue
![image](https://github.com/pjdohertygis/SARCOP/assets/4256979/bfc0b673-8152-46eb-bcb1-d8951fa3e4dc)

### Expected Behavior

We expect the symbols to show up on the map in all cases.

### Reproduction Steps & Sample

It is unclear how to reproduce this bug at this time as it appears to be sporadic but recurring. I'll list some configuration details below. 

1) The feature layer is (and subsequent views are created using the Survey123 Web Form)
2) The Survey123 Web Form does use a reference table for choices from an external feature layer table (but no issues loading this layer) https://napsg.maps.arcgis.com/home/item.html?id=cee151e5ceea4dc6a63b2dc5fd5e7412#overview
3) The feature layer view is using custom symbology by uploading the symbols. 
4) We are using Dashboard URL Parameters to filter the map when embedded in Experience Builder. 
5) There is a "default" state in the Dashboard filter for Deployment vs Training - it defaults to Deployment. 

Hypothesis 1: perhaps 4 and 5 above are conflicting? Test removing the "default" state and see if that fixes the issue since we can control this with URL Parameters now?
![image](https://github.com/pjdohertygis/SARCOP/assets/4256979/64436462-ff40-4607-ad74-114ae8fe9ca3)

### Other Relevant Info

N/A

### Priority impact

p1 - time sensitive

### Impact

This can be extremely confusing for the end user if they cannot see the points on the map.

**Comment by pjdohertygis** [comment]
I tested Hypothesis 1 above - and it does not look like it fixes the issue 

![image](https://github.com/pjdohertygis/SARCOP/assets/4256979/ea36de25-356f-4ce0-9749-43a411443d6e)

Without URL parameter everything loads 
![image](https://github.com/pjdohertygis/SARCOP/assets/4256979/c93e15c2-ff21-48ee-bbfe-01d12dcf438c)

With URL parameter the symbols don't load 
![image](https://github.com/pjdohertygis/SARCOP/assets/4256979/e1edfa40-6541-4413-97e8-73cb79ccaa8b)

**Comment by pjdohertygis** [comment]
Ok I am hitting this issue now in Edge too on my FEMA PC

In ExB 
![image](https://github.com/pjdohertygis/SARCOP/assets/4256979/051acb6b-d963-4dd6-9958-39cf66727234)

Outside ExB 
![image](https://github.com/pjdohertygis/SARCOP/assets/4256979/dcd6aea2-10c9-410d-bacc-60ef33af7964)

**Comment by Jeremy-Bartley** [comment]
@pjdohertygis can you verify that the attribute queries that you are passing in the url are correct from a case sensitivity perspective?  I can't see your map or app to know for sure.  

I see this failing in your screenshots:  https://www.arcgis.com/apps/dashboards/f1bdf94be2c94c8c96e0b8654c7fcd9a#deployment_training=deployment

But if you change it to:  [https://www.arcgis.com/apps/dashboards/f1bdf94be2c94c8c96e0b8654c7fcd9a#deployment_training=**D**eployment](https://www.arcgis.com/apps/dashboards/f1bdf94be2c94c8c96e0b8654c7fcd9a#deployment_training=Deployment)

Does it work?

**Comment by oviera-napsg** [comment]
> @pjdohertygis can you verify that the attribute queries that you are passing in the url are correct from a case sensitivity perspective? I can't see your map or app to know for sure.
> 
> I see this failing in your screenshots: https://www.arcgis.com/apps/dashboards/f1bdf94be2c94c8c96e0b8654c7fcd9a#deployment_training=deployment
> 
> But if you change it to: [https://www.arcgis.com/apps/dashboards/f1bdf94be2c94c8c96e0b8654c7fcd9a#deployment_training=**D**eployment](https://www.arcgis.com/apps/dashboards/f1bdf94be2c94c8c96e0b8654c7fcd9a#deployment_training=Deployment)
> 
> Does it work?

Following @Jeremy-Bartley 's suggestion, here's the original parameter:
![image](https://github.com/pjdohertygis/SARCOP/assets/156027999/f8c2188e-1461-4b49-baa9-2b58dabf9752)

Here's the case-corrected parameter:
![image](https://github.com/pjdohertygis/SARCOP/assets/156027999/77283d6f-de40-4e4c-a2a4-8385d023ea26)

**Comment by oviera-napsg** [comment]
Confirmed that were was also filter conflict between the original Deployment vs Training button filter and the URL parameter.  When a button filter default of **Deployment**  was opened in the Real-World side with a **=Deployment** URL parameter, data displayed correctly.  But the Sandbox side was attempting to pass a **=Training** URL parameter against the default button selection, resulting in a **No data based on current filters** message and no symbology displayed on the map.  I removed the button filter since it is irrelevant to the end-user.  For now, the URL parameters are passing the =Deployment and =Training selections successfully within their respective ExBs.  Since the TFS Dashboard alone isn't user-facing, the fact that all Teams display unfiltered isn't an issue.
